### PR TITLE
Release 10.3.1-rc1

### PR DESCRIPTION
**Pull request by micbar**
## New
- 10.3.1-rc1

## Removed
- 10.2.0 because 10.2.1 is the recommended release, contains security fixes.

### CODE REVIEW

**Comment by tboerger** [comment]
Latest should always be the latest stable release, which an RC cleary is not...

File: .drone.star [hunk 1]
```diff
@@ -27,15 +36,6 @@ def main(ctx):
       'base': 'v18.04',
```
Comment (tboerger) [inline .drone.star:36]: Missing this update

**Review by tboerger — APPROVED** [review]
Feel free to merge and send another pr to fix the base image

File: .drone.star (hunk 1 above)
Comment (micbar) [reply .drone.star:36]: this line affects 10.2.1

Comment (micbar) [reply .drone.star:36]: 10.3.0 already on 19.10

Comment (micbar) [inline .drone.star:36]: this is 10.2.1

Comment (tboerger) [reply .drone.star:36]: Right, bad view of github

**Pull request merged**